### PR TITLE
Organization name changes

### PR DESCRIPTION
**Pull request by jcuanm**
Hardcoded her name onto the alumni page, so the code looks ugly. But I will learn how to make it prettier. She's on there though!

### CODE REVIEW

File: src/Zeega/AdminBundle/Resources/views/edit_custom.html.twig [hunk 1]
```diff
@@ -2,8 +2,12 @@
 
 {% block actions %}
     <div class="sonata-actions">
+<<<<<<< HEAD
```
Comment (horak) [inline src/Zeega/AdminBundle/Resources/views/edit_custom.html.twig]: This little merge conflict snippet should definitely be removed before a merge! Same goes for the change on line 7 and 10.  Also, why was this title changed from "Live Link" to "Archive Link"?

**Comment by horak** [comment]
Thanks @jcuanm, yeah let's use the trans tags just to keep things organized. You can do this properly by adding new Japanese `trans-unit`s to `Messages.Ja.Xliff` [here](https://github.com/horak/Japan-Digital-Archive/blob/waku-integration/src/JDA/CoreBundle/Resources/translations/messages.ja.xliff#L941) and English ones to `Messages.En.Xliff` [here](https://github.com/horak/Japan-Digital-Archive/blob/waku-integration/src/JDA/CoreBundle/Resources/translations/messages.en.xliff#L938).

After you've added those you can then use those trans tags and their associated variable inside `team.html.twig` [here](https://github.com/horak/Japan-Digital-Archive/blob/waku-integration/src/JDA/CoreBundle/Resources/views/Team/team.html.twig#L112) where you might put something like this:

``` php
{% trans %}jda.team.alumni.nakahashi{% endtrans %}<br/>
```

I made some changes very similar to this that you could checkout by studying [this](https://github.com/horak/Japan-Digital-Archive/commit/1deb53fd166d6f95c97abc78319d68e63830ce4f) commit. Maybe @jdaKoko or @meeshmon can give you the proper Japanese characters for all those additions they requested on #988 as well.

Also, very conveniently, if you've made those changes you can close issue #988 by making your commit message something like this: "Organization name changes, closes #988". That will automatically close that issue once I merge the pull request.

**Comment by horak** [comment]
Nice work! There's one other comment I made above you should checkout.

**Comment by jcuanm** [comment]
Thank you! Should I wait for the Japanese characters? 

On Mon, Jun 22, 2015 at 11:16 AM, Alex Horak notifications@github.com
wrote:

> Nice work!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Japan-Digital-Archives/Japan-Digital-Archive/pull/990#issuecomment-114205514
> .

## 

Javier Cuan

**Comment by horak** [comment]
Yeah, @meeshmon can you get us those translations?
- Also, you still need to remove the merge commit comments I mentioned [above](https://github.com/Japan-Digital-Archives/Japan-Digital-Archive/pull/990#discussion-diff-32953967). 
- You've renamed the title to "organization name changes" but haven't included the other name changes  mentioned [here](https://github.com/Japan-Digital-Archives/Japan-Digital-Archive/issues/988).

Work on those two things and hopefully we'll get the translation in the interim!

**Comment by meeshmon** [comment]
Japanese page: http://jdarchive.org/ja/team
From コンテンツ開発 (content development)
ウェブシード　キュレーション (web seed curation)：Kae Nakahashi（昭和ボストン　中橋佳恵）

To チームの卒業者  (Team alumini)
Kae Nakahashi（昭和ボストン　中橋佳恵）: ウェブシード　キュレーション

```
Add to Interaction Design and Development
Project Technical Lead: Alex Horax

プロジェクト技術部リーダー (Project Technical Lead)：Alex Horak（アレックス・ホラック）

Move to "Project Technical Lead: Corinne Curcie" to Team Alumni, "Corinne Curcie:Technical Lead"
```

プロジェクト技術部リーダー：Corinne Curcie（コリン・カーシー）
To チームの卒業者 (Team Alumini)
Corinne Curcie（コリン・カーシー）:プロジェクト技術部リーダー

**Comment by horak** [comment]
Thanks!

**Comment by meeshmon** [comment]
@jcuanm Do you need anything else to finish this up?

**Comment by jcuanm** [comment]
I should be all set. Thank you! I won't be able to look at it until this
Sunday, however.

**Comment by jcuanm** [comment]
I don't quite understand what is happening with the merge conflict. Should I get rid of the highlighted red section code?

**Comment by horak** [comment]
No worries, so when git detects a merge conflict it marks the differences in the files by inserting this ugly thing:

---

<<<<<<< HEAD

code in conflict version from HEAD


more normal code that isn't in conflict

<<<<<<< OTHER-BRANCH-NAME

code in conflict version from OTHER-BRANCH-NAME


---

So yup, you'll need to figure out which branch (HEAD or OTHER-BRANCH-NAME) has the changes you want and edit the files so they only represent the exact changes you want and also pull out all the "<<<"s. 

To avoid merge conflicts always pull before pushing your code.

**Comment by jcuanm** [comment]
I got rid of the code that was conflicting, and the website seems to be operating correctly, along with the changes that were made to the Team page.

**Comment by horak** [comment]
Nice!

**Comment by meeshmon** [comment]
I do not see these changes reflected on the website. Can you guys check again?

**Comment by horak** [comment]
Will do!

**Comment by jcuanm** [comment]
Is this due to the merge conflict?

**Comment by horak** [comment]
Nope we are all set, it's live now!

**Comment by jdaKoko** [comment]
Yumiko Takada's name has be listed under Momoko rather than a single line.
http://jdarchive.org/en/team
yumi s name

Kae's name is both under Japanese content development and team alumni sections in Japanese. 
Please delete her name from the content development.

http://jdarchive.org/ja/team
kae s name is still under the team name

Thank you!

**Comment by jcuanm** [comment]
I'm on it

**Comment by jcuanm** [comment]
I have made the appropriate changes

**Comment by jcuanm** [comment]
I have made the pull-request.

On Thu, Jul 2, 2015 at 10:35 AM, Javier Cuan-Martinez jcuanm@gmail.com
wrote:

> I'm on it
> On Jul 2, 2015 7:09 AM, "jdaKoko" notifications@github.com wrote:
> 
> > Yumiko Takada's name has be listed under Momoko rather than a single line.
> > http://jdarchive.org/en/team
> > [image: yumi s name]
> > https://cloud.githubusercontent.com/assets/8831502/8478808/97576724-20a1-11e5-833c-cdb665b19d41.png
> > 
> > Kae's name is both under Japanese content development and team alumni
> > sections in Japanese.
> > Please delete her name from the content development.
> > 
> > http://jdarchive.org/ja/team
> > [image: kae s name is still under the team name]
> > https://cloud.githubusercontent.com/assets/8831502/8478818/a2933b04-20a1-11e5-8e6b-b7d961df3880.png
> > 
> > Thank you!
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/Japan-Digital-Archives/Japan-Digital-Archive/pull/990#issuecomment-118045733
> > .

## 

Javier Cuan

**Comment by horak** [comment]
Merged and live.

**Pull request merged**